data/strings/strings.txt

```diff
@@ -30687,3 +30687,49 @@
     vi = Lỗi: iCloud không khả dụng
     zh-Hans = 错误：iCloud 不可用
     zh-Hant = 錯誤：iCloud 不可用
+
+  [open_in]
+    comment = Title for the "Open In AppName" buton on the PlacePage.
+    tags = ios
+    en = Open In %@
+    af = Maak oop in %@
+    ar = فتح في %@
+    az = %@ ilə açın
+    be = Адкрыць у %@
+    bg = Отваряне в %@
+    ca = Obre a %@
+    cs = Otevřít v %@
+    da = Åbn i %@
+    de = Öffnen in %@
+    el = Άνοιγμα σε %@
+    es = Abrir en %@
+    et = Avatud %@
+    eu = Ireki %@-n
+    fa = باز کردن در %@
+    fi = Avaa %@
+    fr = Ouvrir en %@
```

Comment:
If the translation "Open In ..." remains, in French it would be more "Ouvrir dans ..." but then if it's the translation "Open with..." (which I think would be better), it would be more like "Ouvrir avec...".

Reply:
@organicmaps/translations-fr  Do you have a different opinion?

Reply:
@Arthur-GYT Thanks! But this in not the final version of the title. Ill mention translators when change the PR from draft to ready for review